competition-simulator/index.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: Competition Simulator
+---
+
+# Competition Simulator
+
+![]({{ site.baseurl }}/resources/competition-simulator/arena.png)
+
+For SR2020, a virtual competition will be run using a brand new simulator.
+
+<div class="info">
+  This is different to the <a href="/docs/programming/simulator/">regular simulator</a>.
+</div>
+
+## Installation
+
+### Prerequisites
+
+This simulation is based in [Webots](https://cyberbotics.com/#download), which will need to be downloaded and installed.
```

Comment:
Given the size of the download it might be worth noting that here. I was initially surprised at the size of it and could imagine that others may think they've got the wrong thing.

Reply:
https://github.com/srobo/docs/pull/122/commits/e2a57e726bb8fdff650f5ae42b7baf7b110e6125